internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,489 @@
+	}, step{
+		PreConfig: func() {
+			w, err := databricks.NewWorkspaceClient(&databricks.Config{})
+			if err != nil {
+				fmt.Println(err)
+			}
+			_, err = w.Lakeview.Update(context.Background(), dashboards.UpdateDashboardRequest{
+				DashboardId:         dashboard_id,
+				DisplayName:         display_name,
+				Etag:                etag,
+				WarehouseId:         warehouse_id,
+				SerializedDashboard: "{\"pages\":[{\"name\":\"b532570b\",\"displayName\":\"New Page Modified Remote\"}]}",
+			})
+			if err != nil {
+				fmt.Println(err)
+			}
+		},
+		Template: makeTemplate(template),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "etag", dashboard.Etag)
+			require.NoError(t, err)
+			return nil
+		}),
+	}, step{
+		Template: makeTemplate(template.SetAttributes(map[string]string{
+			"embed_credentials": "true",
+			"parent_path":       "/Shared/Teams",
+		})),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			require.NoError(t, err)
+			// As the format of the serialized dashboard is not fixed, we can only check if it is not empty
+			assert.NotEqual(t, "", dashboard.SerializedDashboard)
+			return nil
+		}),
+	}, step{
+		PreConfig: func() {
+			os.WriteFile(fileName, []byte("{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page Modified again\"}]}"), 0644)
+		},
+		Template: makeTemplate(template),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			require.NoError(t, err)
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "etag", dashboard.Etag)
+			// As the format of the serialized dashboard is not fixed, we can only check if it is not empty
+			assert.NotEqual(t, "", dashboard.SerializedDashboard)
+			os.Remove("/tmp/LakeviewDashboardTest/Dashboard.json")
+			os.Remove("/tmp/LakeviewDashboardTest")
```

Comment:
Tests may run in parallel:
* Do not remove the folder.
* Either the file or the folder name should contain random characters.
Note that the OS does clean the /tmp folder automatically, so removing the file is ok, but not needed.

Reply:
Got it.